NitroxPatcher/Patches/Stalker_CheckLoseTooth_Patch.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Harmony;
+using NitroxModel.Helper;
+using System;
+using System.Reflection;
+using UnityEngine;
+
+namespace NitroxPatcher.Patches
+{
+    public class Stalker_CheckLoseTooth_Patch : NitroxPatch
+    {
+        public static readonly Type TARGET_CLASS = typeof(Stalker);
+        public static readonly MethodInfo TARGET_METHOD = TARGET_CLASS.GetMethod("CheckLoseTooth", BindingFlags.NonPublic | BindingFlags.Instance);
+
+        public static bool Prefix(Stalker __instance, GameObject target)
+        {
+            float hardness = 0f;
+            TechType techType = CraftData.GetTechType(target);
+            LiveMixin liveMixin = target.GetComponent<LiveMixin>();
+            
+            if (techType == TechType.ScrapMetal || techType == TechType.Titanium
+             || techType == TechType.Seamoth || techType == TechType.Exosuit)
+            {
+                hardness = 0.3f; //15% probability
+            }
+            else if(liveMixin != null && liveMixin.IsAlive())
+            {
+                hardness = 0.1f; //5% probability
```

Comment:
Doesn't unity return a value between 0.0 and 1.0 making anything 0.1 or less a 10% drop chance?  or am I missing something?

Reply:
Same as above, since the probability of dropping is always 50% you have to multiply it by 50% to get the final probability. (5% in the case of alive things)